k8sutils/redis-standalone.go

```diff
@@ -111,3 +112,21 @@ func generateRedisStandaloneContainerParams(cr *redisv1beta1.Redis) containerPar
 	}
 	return containerProp
 }
+
+// generateSidecars generates Redis sidecar informations
+func generateSidecars(cr *redisv1beta1.Redis) []sidecarParameters {
+	sidecars := []sidecarParameters{}
+
+	if cr.Spec.Sidecars != nil {
```

Comment:
I passed over this one but similar to `generateClusterSidecars` I would reverse the conditional here.

Reply:
I've merged both sidecar struct, but I've forgot to push the changes, so this function should been there anymore.